### ISSUE TITLE
Add links for power point presentation and pod cast

### ISSUE THREAD

**Issue by Thrameos** [jpype-project/jpype#883]
@marscher any objections to putting some promotional materials up some where one the site?

We have one set of power point slides one JPype and https://www.pythonpodcast.com/jpype-java-python-bridge-episode-286/  that we can post.   

Not quite sure where to put this sort of material.  I guess I can stuff the pdf can go in the doc section.

[JPype.pdf](https://github.com/jpype-project/jpype/files/5503006/JPype.pdf)

**Comment by marscher** [comment]
not at all, these are very useful informations and should be linked within JPypes documentation. I'm also fine with pulling the pdf in the static files section of sphinx.
